Web Experiences/Blueprints/alert-bar/alertBar.css

```diff
@@ -1,3 +1,12 @@
+/* accomodate every size screen by default */
+/* tested on crome, crome mobile, safari, firefox, edge */
+
+/*
+Declare variables
```

Comment:
I think we could make this a bit more explicit "Declare Boxever Template Variables". People might be like "what is this weird syntax?!"

Reply:
Done